.github/workflows/test.yml

```diff
@@ -36,9 +36,18 @@ jobs:
       matrix:
         include:
           - container: ghcr.io/oca/oca-ci/py3.6-odoo14.0:latest
+            include: "product_expiry_simple,product_expiration_date"
             makepot: "true"
             name: test with Odoo
           - container: ghcr.io/oca/oca-ci/py3.6-ocb14.0:latest
+            include: "product_expiry_simple,product_expiration_date"
+            name: test with OCB
+          - container: ghcr.io/oca/oca-ci/py3.6-odoo14.0:latest
+            exclude: "product_expiry_simple,product_expiration_date"
+            makepot: "true"
+            name: test with Odoo
+          - container: ghcr.io/oca/oca-ci/py3.6-ocb14.0:latest
+            exclude: "product_expiry_simple,product_expiration_date"
             name: test with OCB
```

Comment:
Please @rousseldenis could you help to fix configuration to get it works
by comparison of https://github.com/OCA/server-tools/blob/14.0/.github/workflows/test.yml#L36
it seems ok, but in real not

Reply:
Yes, I saw your tries.

But, the problem here I think is the mutual exclusion in the manifest. I think the product_expiry module is installed in the first step as required by other modules.

I think you should exclude them also and put in the rebel modules: stock_mass_scrap, stock_move_quick_lot. Could you try ?

@sbidoul Any advice?

Reply:
Looking at the error I see "product_expiration_date does not exist". But indeed this module does not exist in this repo?

Reply:
You're right, madre mia, I shouldn't work after 22h, caramba !